src/main/scala/centaur/test/Test.scala

```diff
@@ -67,7 +66,8 @@ object Operations {
         // Collect only the parameters which exist:
         val params = List("wdlSource" -> Option(workflow.data.wdl),
           "workflowInputs" -> workflow.data.inputs,
-          "workflowOptions" -> workflow.data.options) collect { case (name, Some(value)) => (name, value) }
+          "workflowOptions" -> WorkflowOptions(workflow.data.options).insertSecrets.options
```

Comment:
I hear you--and my thought was that it's potential that with future tests may require tweaking of workflow options. Otherwise, I'm totally okay with just adding the insertSecrets method as a def in Test.scala. @mcovarr not sure if that was a TOL or you feel strongly one way?